app/code/Magento/Wishlist/Controller/Shared/Allcart.php

```diff
@@ -3,55 +3,62 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
+declare(strict_types=1);
+
 namespace Magento\Wishlist\Controller\Shared;
 
+use Magento\Framework\App\Action\Action;
 use Magento\Framework\App\Action\Context;
-use Magento\Wishlist\Model\ItemCarrier;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\Controller\Result\Forward;
+use Magento\Framework\Controller\Result\Redirect;
 use Magento\Framework\Controller\ResultFactory;
+use Magento\Wishlist\Model\ItemCarrier;
 
-class Allcart extends \Magento\Framework\App\Action\Action
+class Allcart extends Action implements HttpGetActionInterface
 {
     /**
-     * @var WishlistProvider
+     * @var ItemCarrier
      */
-    protected $wishlistProvider;
+    private $itemCarrier;
 
     /**
-     * @var \Magento\Wishlist\Model\ItemCarrier
+     * @var WishlistProvider
      */
-    protected $itemCarrier;
+    private $wishlistProvider;
 
     /**
      * @param Context $context
-     * @param WishlistProvider $wishlistProvider
      * @param ItemCarrier $itemCarrier
+     * @param WishlistProvider $wishlistProvider
      */
     public function __construct(
         Context $context,
-        WishlistProvider $wishlistProvider,
-        ItemCarrier $itemCarrier
+        ItemCarrier $itemCarrier,
+        WishlistProvider $wishlistProvider
     ) {
-        $this->wishlistProvider = $wishlistProvider;
         $this->itemCarrier = $itemCarrier;
+        $this->wishlistProvider = $wishlistProvider;
```

Comment:
Please revert this change
```suggestion
        $this->wishlistProvider = $wishlistProvider;
        $this->itemCarrier = $itemCarrier;
```